.github/workflows/ci.yaml

```diff
@@ -73,20 +73,6 @@ jobs:
         run: |
```

Comment:
maybe rename workflow to build?

Reply:
Yeah, naming is bad still

Reply:
As with the previous comment, I removed the new file, commented here: https://github.com/lambdaclass/lambda_ethereum_rust/pull/1032#issuecomment-2450670401